### ISSUE TITLE
i2 Analyse system match rules

### ISSUE THREAD

**Issue by sdorresteijn88** [i2group/analyze#76]
Good morning, 
 
I am playing around with the system match rules to prevent duplicates in the information store. However, I don’t get a warning when I upload 2 ‘the same’ vehicles to the information store. Can you see what I do wrong?
  
1.	I use i2 Analyze 4.4.2, i2 Analyze containers 2.6.0 and ANBP 9.4
2.	I changed the match rules to the example described here, [https://docs.i2group.com/analyze/4.4.2/r_matching_rules_config.html].
3.	I deployed the config again.
4.	I added manual 3 cars on a chart, with the License Plate ABC123, ABC 123 and ABC-123.
![image](https://github.com/i2group/analyze/assets/52672945/e71e6522-646b-4873-b146-d5caab3890ae)
5.	I uploaded ABC123 to the info store, and after this the ABC 123. I expected a warning that there is a duplicate. But I didn’t get a warning and both cars were added to the information store.
    
Below the content of the system-match-rules.xml
```
<?xml version="1.0"?>
<tns:matchRules xmlns:tns="http://www.i2group.com/Schemas/2019-05-14/MatchRules"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema"
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://www.i2group.com/Schemas/2019-05-14/MatchRules MatchRules.xsd"
  version="2"
  enableSourceIdentifierMatching="true">
    <matchRule id="4a2b9baa-e3c4-4840-a9fd-d204711af50e"
      itemTypeId="ET3"
      displayName="Match vehicles"
      description="Match vehicles with the same license plate,
                  when either the registered state or
                  region are the same."
      active="true">
      <matchAll>
        <!-- License Plate Number -->
        <condition propertyTypeId="VEH2">
          <operator>EXACT_MATCH</operator>
          <normalizations>
            <normalization>IGNORE_WHITESPACE_BETWEEN</normalization>
          </normalizations>
        </condition>
      </matchAll>
      <matchAny>
        <!-- Registered State -->
        <condition propertyTypeId="VEH16">
        <operator>EXACT_MATCH</operator>
          <normalizations>
            <normalization>IGNORE_CASE</normalization>
            <normalization>IGNORE_DIACRITICS</normalization>
            <normalization>IGNORE_WHITESPACE_BETWEEN</normalization>
            <normalization>IGNORE_NONALPHANUMERIC</normalization>
          </normalizations>
        </condition>
        <!-- Registered Country -->
        <condition propertyTypeId="VEH15">
          <operator>EXACT_MATCH</operator>
          <normalizations>
            <normalization>IGNORE_CASE</normalization>
            <normalization>IGNORE_DIACRITICS</normalization>
            <normalization>IGNORE_WHITESPACE_BETWEEN</normalization>
            <normalization>IGNORE_NONALPHANUMERIC</normalization>
          </normalizations>
        </condition>
      </matchAny>
    </matchRule>
</tns:matchRules>
```

**Comment by sdorresteijn88** [comment]
Hi @Colvin-Cowie-i2  thanks for your quick response! I'll raise a RFE via support.
